src/main/scala/io/iohk/ethereum/consensus/atomixraft/AtomixRaftMiner.scala

```diff
@@ -18,26 +18,26 @@ import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.util.{Failure, Success}
 
-class Miner(
+class AtomixRaftMiner(
```

Comment:
As noted on the [previous PR](https://github.com/input-output-hk/mantis/pull/407#discussion_r171842862) _mining_ is strictly related to PoW, whereas _forging_ (_forge_, _forger_) is a more general term. Can we replace the references to _mining_ within the context of Raft? If, through abstraction, _forging_ leaks to Ethash, that's fine too.

Reply:
Yes, as I have explained, "miner" and "mining" is used very loosely/liberally in our non-PoW use-cases. This was in the absense of a better name and since naming is hard I did not want to spend more cycles on it :)

Happy to discuss and find a more appropriate terminology though. Forging sounds cool. Is it used anywhere else or are we the first ones to do it?

Reply:
Some examples:
https://cardanodocs.com/glossary/#follow-the-satoshi
https://medium.com/@karthik.seshu/cryptocurrency-proof-of-work-vs-proof-of-stake-e1eee1420b10
http://www.cryptomorrow.com/2017/10/23/staking-wallet/
(just text-search for "forge")

Now, I don't think this term has been officially dubbed, but I've seen it used many times in the context. Another word that appears is _minting_, but I think it's less common.

Reply:
I thought about minting as well but yes, it does not seem to be widely used.

On second thought, `forge` may lead to [negative connotations](https://www.merriam-webster.com/dictionary/forger), so I am not sure it is a good idea anymore.

Reply:
I think that's a secondary meaning, so I wouldn't worry about that. But it might be best to ask a native speaker, @mcsherrylabs ?

Reply:
Wow. Much intricacy. `Minting` almost works, but is associated specifically with coin. I like `forge` and for a native speaker it's very clear which meaning is being used. Of course not all readers will be native speakers. It's a good point that `AtomixRaftForger` or just seeing `forger` in the code might cause the reader to pause and scratch their head. :thinking: ..... not going to overthink this. I think `forge` is the best candidate. The root cause of the confusion is the English language and that's out of scope.

Reply:
`RaftForger` [it is](https://github.com/input-output-hk/mantis/pull/410/commits/8c5433f2145e75a6c95ac9c9f7199bd67f6b7aac)!